mitosheet/mitosheet/excel_utils.py

```diff
@@ -0,0 +1,47 @@
+
+
+
+import string
+from typing import Tuple
+
+
+def get_excel_range_from_column_index(col_index: int) -> str:
+    """
+    Number to Excel-style column name, e.g., 1 = A:A, 26 = Z:Z, 27 = AA:AA, 703 = AAA:AAA.
+    """
+    return get_column_from_column_index(col_index) + ":" + get_column_from_column_index(col_index)
+
+def get_column_from_column_index(col_index: int) -> str:
+    """
+    Number to Excel-style column name, e.g., 1 = A, 26 = Z, 27 = AA, 703 = AAA.
+    """
+    # Add 1 because Mito 0 indexes columns
+    col_index = col_index + 1
+    name = ''
+    while col_index > 0:
+        col_index, r = divmod (col_index - 1, 26)
+        name = chr(r + ord('A')) + name
+    return name
+
+def get_row_and_col_indexes_from_range(range: str) -> Tuple[Tuple[int, int], Tuple[int, int]]:
+    start_cell_address, end_cell_address = range.split(':')
```

Comment:
We should probably handle errors here or in the helper function if the range does not have a `:` in it.